### PR TITLE
New package: swww-0.9.5

### PR DESCRIPTION
**Pull request by Calandracas606**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl

### CODE REVIEW

**Comment by classabbyamp** [comment]
#43441

**Comment by classabbyamp** [comment]
I think your template might be more correct (e.g. the makedepends)

**Comment by Calandracas606** [comment]
bump

**Comment by tranzystorekk** [comment]
LGTM

**Comment by Calandracas606** [comment]
submitted patch upstream: https://github.com/LGFae/swww/pull/254

File: srcpkgs/swww/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+version=0.9.5
+revision=1
+build_style=cargo
+hostmakedepends="scdoc pkg-config"
+makedepends="liblz4-devel"
+short_desc="Solution to your Wayland Wallpaper Woes"
+maintainer="Daniel Martinez <danielmartinez@cock.li>"
+license="GPL-3.0-only"
+homepage="https://github.com/LGFae/swww"
+changelog="https://raw.githubusercontent.com/LGFae/swww/v${version}/CHANGELOG.md"
```
Comment (classabbyamp) [inline srcpkgs/swww/template:12]: use default branch for changelog, not versioned

Comment (Calandracas606) [reply srcpkgs/swww/template]: done

**Pull request merged**